internal/compression/keys.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compression
+
+// Compression keys for supported compression types within opencensus collector
+const (
+	Unsupported = "gzip"
```

Comment:
Oh oh, was this a copy+paste mistake? Making Unsupported be "gzip"?

Reply:
:o idk how i did that. Definitely a typo

Reply:
its supposed to be empty string

Reply:
good catch

Reply:
Cool, good catch!